app/javascript/mastodon/components/status.jsx

```diff
@@ -332,13 +335,8 @@ class Status extends ImmutablePureComponent {
     this.handleToggleMediaVisibility();
   };
 
-  handleUnfilterClick = e => {
-    this.setState({ forceFilter: false });
-    e.preventDefault();
-  };
-
-  handleFilterClick = () => {
-    this.setState({ forceFilter: true });
+  handleFilterToggle = () => {
+    this.setState({ forceFilter: !this.state.forceFilter });
```

Comment:
It might be cleaner to use something like `this.setState(state => ({ forceFilter: !state.forceFilter}))`. This would be better with hooks, but I dont remember if class components have the same issues here.